Doc/library/pdb.rst

```diff
@@ -288,7 +288,7 @@ There are three preset *convenience variables*:
 * ``$_retval``: the return value if the frame is returning
 * ``$_exception``: the exception if the frame is raising an exception
 
-.. versionadded:: 3.12
+.. versionadded:: 3.12  *convenience variable*
```

Comment:
I think this is how we'd normally do this kind of thing:

```suggestion
.. versionadded:: 3.12

   Added the three convenience variables
```

E.g. see https://github.com/python/cpython/blob/5d04cc50e51cb262ee189a6ef0e79f4b372d1583/Doc/library/typing.rst?plain=1#L891-L897

Reply:
Okay I copied the format from `enum.rst` where they do thing differently...

Reply:
Also this is what confused the user - we did not only add the three convenience variables, we added the convenience variable feature, together with three pre-defined convenience variables.

Reply:
> Okay I copied the format from `enum.rst` where they do thing differently...

Oh, fair enough!

> Also this is what confused the user - we did not only add the three convenience variables, we added the convenience variable feature, together with three pre-defined convenience variables.

I see. I think your proposed text is maybe slightly too terse in that case; I still misread it as a non-pdb expert